.github/workflows/liveness-branch-sync.yml

```diff
@@ -0,0 +1,21 @@
+name: Liveness Release Branch Sync
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  branch-sync:
+    name: Branch Sync
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Sync branches
+        uses: repo-sync/pull-request@v2
```

Comment:
Technically, we might want some security discussion before we use 3p actions. We can sync offline!